linebot/api.py

```diff
@@ -1148,6 +1149,244 @@ def get_bot_info(self, timeout=None):
 
         return BotInfo.new_from_json_dict(response.json)
 
+    def create_audience_group(self, audience_group_name, audiences=[],
+                              is_ifa=False, timeout=None):
+        """Create an audience group.
+
+        https://developers.line.biz/en/reference/messaging-api/#create-upload-audience-group
+
+        :param str audience_group_name: The audience's name
+        :param list audiences: An array of user IDs or IFAs
+        :param bool is_ifa: true | false
+        :return: audience group id
+        """
+        response = self._post(
+            '/v2/bot/audienceGroup/upload',
+            data=json.dumps({
+                "description": audience_group_name,
+                "isIfaAudience": is_ifa,
+                "audiences": [{"id": audience} for audience in audiences[:10000]],
+            }),
+            timeout=timeout
+        )
+        audience_group_id = response.json.get('audienceGroupId')
+        if 10000 < len(audiences):
+            self.add_audiences_to_audience_group(audience_group_id, audiences[10000:])
+
+        return audience_group_id
+
+    def get_audience_group(self, audience_group_id, timeout=None):
+        """Get the object of audience group.
+
+        https://developers.line.biz/en/reference/messaging-api/#get-audience-group
+
+        :param str audience_group_id: The audience ID
+        :param timeout: (optional) How long to wait for the server
+            to send data before giving up, as a float,
+            or a (connect timeout, read timeout) float tuple.
+            Default is self.http_client.timeout
+        :type timeout: float | tuple(float, float)
+        :return: AudienceGroup instance
+        """
+        response = self._get(
+            '/v2/bot/audienceGroup/{audience_group_id}'.format(
+                audience_group_id=audience_group_id),
+            timeout=timeout
+        )
+        result = dict(jobs=response.json.get('jobs', []), **response.json.get('audienceGroup', {}))
```

Comment:
I do not understand why you create a new dist?
Because other API just input `response.json` and it will get the return values.
https://github.com/line/line-bot-sdk-python/blob/master/linebot/api.py#L301

Reply:
modified